accounts/models.py

```diff
@@ -14,12 +14,6 @@
 
 class User(AbstractUser):
     username = models.CharField(max_length=64, default=None)
```

Comment:
You should set `username=None` because this previous commit broken it

https://github.com/GeeksCAT/anem-per-feina/commit/0db43476a3e1eb02b32952e4d9b7655b8e258024#diff-6b070a300f0e440e2b6e3ae75cd4a54300517cbc9bdbb9a08584f10c026abaa1